stl/inc/format

```diff
@@ -314,6 +314,8 @@ public:
     using iterator       = const_iterator;
 

```

Comment:
There's a `// TODO: test coverage` comment above this class definition - should it be removed now?